File: dart/baby-steps/person.dart
class Person {
  String name;
  int age;

  Person(this.name, this.age) {}

  sayHi() {
    print("Hi! my name is $name");
  }
}

void main(List<String> args) {
  var person = {
    "name": "Luis",
    "age": 21,
  };
  print(person);
  print(person["name"]);

  var personInstance = new Person("Luis", 21);
  personInstance.sayHi();
}

File: kotlin/baby-steps/person.kt
class Person(val name: String, val age: Int) {
  fun sayHi() {
    println("Hi! my name is $name");
  }

  fun sayAge() {
    println("I'm $age years old");
  }
}

fun main() {
  val person = mapOf("name" to "Luis", "age" to 21);
  println(person);
  println(person["name"]);

  val personInstance = Person("Luis", 21);
  personInstance.sayHi();
  personInstance.sayAge();
}
